examples/showcase/instancing/index.html

```diff
@@ -5,7 +5,7 @@
   import {WebGLDevice} from '@luma.gl/webgl';
   import {WebGPUDevice} from '@luma.gl/webgpu';
   import AnimationLoopTemplate from './app.ts';
-  luma.registerDevices([WebGLDevice]);
+  luma.registerDevices([WebGPUDevice]);
```

Comment:
Does this example take advantage of increased limits?

Reply:
No I don't have any example that need that many attributes. I believe that is your area of expertise :)

I just switched this example to WebGPU to test the code path and inspect the limits, haven't actually tested in